user_guide_src/source/changelogs/v4.6.0.rst

```diff
@@ -66,6 +66,9 @@ Removed Deprecated Items
 Enhancements
 ************
 
+- Added `retainMultiplePatterns()` to `FileCollection` class.
```

Comment:
You are adding it in the wrong place.
`FileCollection` is a library, so please add in the [`Libraries` section](https://github.com/codeigniter4/CodeIgniter4/blob/61dc880edc560ead12a28a3713160043930f2747/user_guide_src/source/changelogs/v4.6.0.rst#libraries}